components/navigation/navigation.css

```
@@ -0,0 +1,22 @@
+.navigation {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
```

You could write it like that:
```suggestion
  grid-template-columns: repeat(4, 1fr);
```